README.md

```diff
@@ -1,3 +1,16 @@
 # Abtion Guidelines
 
 Please, go to the specific folder for more info.
+
+
+###Staging environment
+
+**Developing the project:** No need for staging. As no one is using the project, there can (but should not) be "bugs" in production.
+  
+**Non-actively develop project:** No need for permanent staging environment. Whenever there is a task that needs to be tested by the client, a temporal staging environment should be spin up.
```

Comment:
should be **spun** up.
spin spun spun... I think _span_ works if you feel oldschool.